source/upgrade.txt

```diff
@@ -111,8 +111,14 @@ Version 4.11 Breaking Changes
 
 - {+mdb-server+} v3.6 is no longer supported. The minimum supported {+mdb-server+}
   version is now v4.0.
-
-- The minimum wire version is now 7. See :manual:`minWireVersion </reference/command/hello/#mongodb-data-hello.minWireVersion>`.
+- Python v3.8 is no longer supported. The minimum supported Python version is
+  now v3.9.
+- PyPy v3.9 is no longer supported. The minimum supported PyPy version is now
+  v3.10.
+- The minimum wire version is now 7. For more information, see
```

Comment:
s: v7?  or 7.0?

Reply:
Good question, this was already in here from a previous ticket but I don't see it on the release notes. I'll have this double-checked by the tech reviewer

Reply:
Users don't understand wire version and we already say `v3.6 is no longer supported.` above. I suggest we remove the wire version note.